src/components/List.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+// Components
+import Card from './Card';
+
+function List(props) {
+  if (props.data.length == 0) {
+    return (
+      <div className="panel__list">
+        <div className="list__card">
+          <p className="list__disclaimer">No Devices</p>
+        </div>
+      </div>
+    )
+  }
+
+  return (
+    <div className="panel__list">
+      {props.data.map((device, index) => (
+        <Card
+          key={index}
+          id={device.id}

```

Comment:
This Card component could be improved in a couple of ways. It's coupled really tightly to the `Device` model (since the property names are the same as the Device properties) and there are just a lot of properties in general. A better pattern might be to think of the Card as a reusable component with its own independent "content slots", e.g.

```
<Card heading="" subhead="" description="" />
```

You can still map the properties of Device to these, by doing something like this:

```
<Card 
    key={device.id}
    heading={`${device.manufacturer} ${device.model}`}
    subhead={`${device.platform} ${device.version}`}
    description={`Serial #: ${device.platform}`}
/>
```

You'll notice that I swapped the value of `key`; if you have a unique identifier (like device.id) in your data model, it's always better to use that instead of `index`.